jekyll/_cci2/server/installation/phase-1-prerequisites.adoc

```diff
@@ -0,0 +1,821 @@
+        "s3:GetLifecycleConfiguration",
+        "s3:GetInventoryConfiguration",
+        "s3:GetBucketTagging",
+        "s3:PutAccelerateConfiguration",
+        "s3:DeleteObjectVersion",
+        "s3:GetBucketLogging",
+        "s3:ListBucketVersions",
+        "s3:ReplicateTags",
+        "s3:RestoreObject",
+        "s3:ListBucket",
+        "s3:GetAccelerateConfiguration",
+        "s3:GetBucketPolicy",
+        "s3:PutEncryptionConfiguration",
+        "s3:GetEncryptionConfiguration",
+        "s3:GetObjectVersionTorrent",
+        "s3:AbortMultipartUpload",
+        "s3:PutBucketTagging",
+        "s3:GetBucketRequestPayment",
+        "s3:GetAccessPointPolicyStatus",
+        "s3:GetObjectTagging",
+        "s3:GetMetricsConfiguration",
+        "s3:PutBucketVersioning",
+        "s3:GetBucketPublicAccessBlock",
+        "s3:ListBucketMultipartUploads",
+        "s3:PutMetricsConfiguration",
+        "s3:PutObjectVersionTagging",
+        "s3:GetBucketVersioning",
+        "s3:GetBucketAcl",
+        "s3:PutInventoryConfiguration",
+        "s3:GetObjectTorrent",
+        "s3:PutBucketWebsite",
+        "s3:PutBucketRequestPayment",
+        "s3:PutObjectRetention",
+        "s3:GetBucketCORS",
+        "s3:GetBucketLocation",
+        "s3:GetAccessPointPolicy",
+        "s3:GetObjectVersion",
+        "s3:GetAccessPoint",
+        "s3:GetAccountPublicAccessBlock",
+        "s3:ListAllMyBuckets",
+        "s3:ListAccessPoints",
+        "s3:ListJobs"
+      ],
+      "Resource": [
+        "arn:aws:s3:::<YOUR_BUCKET_NAME>",
+        "arn:aws:s3:::<YOUR_BUCKET_NAME>/*"
+      ]
+    }
+  ]
+}
+----
+
+. Attach policy to user.
++
+[source,shell]
+----
+aws iam put-user-policy \
+  --user-name circleci-server \
+  --policy-name circleci-server \
+  --policy-document file://policy.json
+----
+
+. Create Access Key for user circleci-server.
++
+NOTE: You will need this when you configure your server installation later.
++
+[source,shell]
+----
+aws iam create-access-key --user-name circleci-server
+----
++
+The result should look like this:
++
+[source, json]
+----
+{
+  "AccessKey": {
+        "UserName": "circleci-server",
+        "Status": "Active",
+        "CreateDate": "2017-07-31T22:24:41.576Z",
+        "SecretAccessKey": <AWS_SECRET_ACCESS_KEY>,
+        "AccessKeyId": <AWS_ACCESS_KEY_ID>
+  }
+}
+----
+
+endif::env-gcp[]
+
+ifndef::env-aws[]
+
+[#google-cloud-storage]
+=== Google Cloud Storage
+
+[#create-a-gcp-bucket]
+==== a. Create a GCP bucket
+If your server installation runs in a GKE cluster, ensure that your current IAM user is cluster admin for this cluster, as RBAC (role-based access control) objects need to be created. More information can be found in the https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control[GKE documentation].
+
+[source,shell]
+----
+gsutil mb gs://circleci-server-bucket
+----
+
+[#set-up-authentication]
+==== b. Set up authentication
```

Comment:
This section has the same name "Set up authentication" which causes the ToC nav to show the user is on both of them (they are both highlighted blue). Not an issue for here, but probably something we should bring up with Docs Disco to see if we can fix that. It's reasonable to think that some sections will have the exact same names, particularly when a user needs to follow the same steps for different services.